### DISCUSSION TITLE
Any plan update to support mac m1 device?

### DISCUSSION THREAD

**Discussion by linhanyu** [eclipse-theia/theia#9791] [General] [answered]
Our team use theia to develop a electron app. Most of crash occurred in m1 device.

**Comment by linhanyu** [comment, marked as answer]
We trying update our product to electron11, but stuck at some native npm package, such as theia/node-pty, drivelist. (also, compile in m1 devices were very slow).

**Reply by vince-fugnitto** [reply]
@linhanyu I'm sure in the future there will plans to better support the M1 CPU, unfortunately I do not believe main community has a device yet to perform proper support. If you can help it'd be greatly appreciated of course.

As for Electron 11, we have the following issue to track the feature https://github.com/eclipse-theia/theia/issues/9095, and a pull-request as well which attempts to perform the upgrade https://github.com/eclipse-theia/theia/pull/9096. It hasn't been active for a while, if you're interested you can also help out.
